### PR TITLE
A Valorian Language (and a ticker fix)

### PR DESCRIPTION
**Pull request by lavillastrangiato**
## About The Pull Request

- Adds New Valorian, mostly made by Laughing Rabbit.
- You can't start the game without a single person anymore.

## Why It's Good For The Game

I wanted the first thing, some people wanted the second thing for some reason.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -300,31 +300,15 @@ SUBSYSTEM_DEF(ticker)
...
-			ikon = new(fcopy_rsc(file_path))
-			if(SStitle.splash_turf && ikon)
-				SStitle.splash_turf.icon = ikon
-			for(var/mob/dead/new_player/player in GLOB.player_list)
-				player.playsound_local(player, 'sound/music/wartitle.ogg', 100, TRUE)*/
+	if(amt_ready < amt_ready_needed)
+		to_chat(world, "<span class='purple'>Not enough players to start the game.</span>")
 	job_change_locked = TRUE
-	return TRUE
+	return FALSE // If you ever want to stop the game from starting without a ruler, then just uncomment that block of code up there. Right now, this stops the game from starting if there's only one person readied.
```
Comment (BadAtThisGame302) [inline code/controllers/subsystem/ticker.dm:311]: You deleted the block of code up there, would be hard to uncomment it.

**Comment by BadAtThisGame302** [comment]
What's the font for the new language?

**Comment by lavillastrangiato** [comment]
@BadAtThisGame302 It doesn't have one but I can add one when I get home/this week, maybe

File: code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (lavillastrangiato) [reply code/controllers/subsystem/ticker.dm:311]: image

**Comment by lavillastrangiato** [comment]
stop giving me SHIT you stupid linter

**Pull request closed without merging**